### DISCUSSION TITLE
The AJAX search popup is not working when specific customer pages are excluded from the Breeze theme.

### DISCUSSION THREAD

**Discussion by prachi-cp** [breezefront/community#300] [1. Help]
when specific customer pages like (customer/account/) are excluded from the Breeze theme, then which is causes the AJAX search popup to malfunction(Swissup_Ajaxsearch).

Could you please provide the correct way to open the AJAX popup without breeze?

**Comment by vovayatsyuk** [comment]
Do you mean you cannot activate search input on mobile device? Just fixed this issue in swissup/module-ajaxsearch 1.13.19

**Reply by prachi-cp** [reply]
I have added a custom URL in Disable Breeze for the specified URLs section but the search functionality is not working for that URL... Can you please explain how to open the AJAX popup when excluding the URL?
![exclud_url](https://github.com/user-attachments/assets/9471110e-2d6c-4510-b501-a8714af3967c)

**Reply by vovayatsyuk** [reply]
Please provide the URL to the page where the search is not working.
